src/parse-nm.c

```diff
@@ -386,6 +386,10 @@ parse_dot1x_auth(GKeyFile* kf, NetplanAuthenticationSettings* auth)
             auth->eap_method = NETPLAN_AUTH_EAP_PEAP;
         } else if (g_strcmp0(first_method, "ttls") == 0) {
             auth->eap_method = NETPLAN_AUTH_EAP_TTLS;
+        } else if (g_strcmp0(first_method, "leap") == 0) {
+            auth->eap_method = NETPLAN_AUTH_EAP_LEAP;
+        } else if (g_strcmp0(first_method, "pwd") == 0) {
```

Comment:
thought: We probably want some `else` case here, for any other (future) method, unknown by Netplan. Those should be handled via the `networkmanager.passthrough` setting.

As you mentioned there are more, e.g. "md5", "fast", ...

Reply:
Okay, it seems like the generic/fallback case is a bit more involved. Running into specific edge-cases. We will handle that in a separate PR.